frontend/src/ExploreGensApp.vue

```diff
@@ -199,8 +199,8 @@ export default {
       this.items = await this.rpc.invoke("getRecommendedQuery");
       this.items.unshift(ALL_GENS);
     },
-    async setIsInTheia() {
-      this.isInTheia = await this.rpc.invoke("isInTheia");
+    async setIsInBAS() {
+      this.isInBAS = await this.rpc.invoke("getInBAS");
```

Comment:
getIsInBAS

Reply:
done

Reply:
async setIsInBAS() {
      this.isInBAS = await this.rpc.invoke("getIsInBAS");